Modules/_winapi.c

```diff
@@ -1553,6 +1553,28 @@ _winapi_ReadFile_impl(PyObject *module, HANDLE handle, DWORD size,
     return Py_BuildValue("NI", buf, err);
 }
 
+/*[clinic input]
+_winapi.ResumeThread
```

Comment:
I don't see the point in throwing away the thread's previous suspend count when it's so easy to just return it. I think you should change this line of the definition to `_winapi.ResumeThread -> DWORD`, and return `result`. It could be useful later on for some other problem.